.github/workflows/haskell.yml

```diff
@@ -115,7 +115,10 @@ jobs:
         TMPDIR: ${{ runner.temp }}
         TMP: ${{ runner.temp }}
         KEEP_WORKSPACE: 1
-      run: cabal test all
+      run: |
+        cabal test all
+        rm -rf cardano-cli/test/cardano-cli-golden/files/golden
```

Comment:
```suggestion
        rm -Rf cardano-cli/test/cardano-cli-golden/files/golden
```

Reply:
aren't `-r` and `-R` equivalent?

Reply:
The CI seems not to like `-r` :smile_cat: 

![image](https://github.com/input-output-hk/cardano-cli/assets/634720/85be5f24-b8eb-4cd7-a814-83f80e71d7b3)

Reply:
oh that's powershell, and not bash 🤦🏻